### ISSUE TITLE
Pylance can't analyze base module but can analyze sub-modules

### ISSUE THREAD

**Issue by zbs** [microsoft/pylance-release#853]
`from foo.bar import baz` is analyzed (e.g. `foo` and `bar` components can be CTRL+clicked), but the following cannot:

`from foo import bar`
`import foo`

My workspace structure is*
```
root
-- foo
---- foo
------ __init__.py
------ bar
-------- __init_.py
-------- baz
-- other_packages
-- main
---- test.py
```

and in test.py:
```
import foo
```

My `python.analysis.extraPaths`looks like:
```
python.analysis.extraPaths = [
  "foo",
]
```

My Pylance version is 2021.1.1 and my VS Code version is 1.52.1.

*Note that this is an abstract representation of my workspace structure, not my actual workspace structure.

**Comment by erictraut** [comment]
Thanks for the bug report. The issue here is that you have two nested `foo` directories. Pylance is resolving the `foo` import to the top-level `foo` directory when it should be selecting the nested one.

I've fixed the underlying problem, and it will be included in the next release of pylance.

**Comment by zbs** [comment]
Thanks for the quick response! Looking forward to it.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.1.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202112-20-january-2021
